packages/nimble-components/src/utilities/models/converter.ts

```diff
@@ -0,0 +1,17 @@
+import { booleanConverter, ValueConverter } from '@microsoft/fast-element';
+
+export const optionalBooleanConverter: ValueConverter = {
```

Comment:
FAST actually has a nullableBooleanConverter, and the implementation looks very similar to this. Can you double check that the existing one in FAST isn't sufficient for this case?

https://github.com/microsoft/fast/blob/218cfc014f5c76464493725f83bd883e8b07defa/packages/web-components/fast-element/src/components/attributes.ts#L96

Reply:
You've linked to the `nullableNumberConverter`. I do see that there is a `nullableBooleanConverter` in the main branch, but I don't see it in the branch that we use. Anyway, theirs wouldn't work for us because we need setting to `undefined` not to convert to `null`.

Reply:
Why is the distinction between `null` and `undefined` important here? The value given to the formatter will ultimately be `true`, `false`, or `undefined`, right?

Reply:
Actually, I guess you're right. We pass `this.customHour12 ?? undefined`, so even if the property gets a `null` value, we'll pass `undefined` to the formatter. Still, since there is no `nullableBooleanConverter` in the version of FAST that we use, we have to define our own, and it seems reasonable (to me) to have it pass `null` and `undefined` through unchanged. Is there an argument for doing it a different way?

Reply:
Will the `nullableBooleanConverter` be available when we complete #1427? If so, we should have a note somewhere that this converter should be replaced with that one once it is available.

Reply:
It still isn't available. I've added a comment reminding us to adopt the FAST converter when it's available.